portfolio/src/main/java/com/google/sps/data/CuisineDatastore.java

```diff
@@ -0,0 +1,111 @@
+package com.google.sps.data;
+
+import java.util.Map;
+import java.util.HashMap;
+import com.google.appengine.api.datastore.*;
+import com.google.common.collect.ImmutableMap;
+import com.google.sps.data.Constants;
+import com.google.sps.data.Constants.CuisineEnum;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Class that handles writing/reading cuisine data from the datastore. */
```

Comment:
this file has too many issues related to how you are mixing enum, string and such.  i let you review this first.